src/routes/+page.svelte

```diff
@@ -0,0 +1,93 @@
+<script lang="ts">
+    import { onMount } from 'svelte';
+    import Gallery from '../components/Gallery.svelte';
+    import MovieDetail from '../components/MovieDetail.svelte';
+    import NavBar from '../components/NavBar.svelte';
+    import '/src/global.css';
+    import type { Movie } from '../interfaces/Movie';
+    import { page } from '$app/stores';
+    import { goto } from '$app/navigation';
+
+    let movies: Movie[] = [];
+    let currentIndex = -1;
+    let showDetail = false;
+    let currentMovie: Movie | undefined;
+    let franchiseId: number;
+    let error: string | null = null;
+
+    $: {
+        const { movies: loadedMovies } = $page.data;
+        if (loadedMovies) {
+            movies = loadedMovies;
+            error = null;
+        } else {
+            error = "No movies loaded";
+            console.log(error);
```

Comment:
```suggestion
```

Reply:
Your eyes are better then mine! I didn't see it removed it now:

https://github.com/BowenGeense/MovieShowcase/pull/3/files#diff-bfe0e412488effea14eec1aad806e9297fb0b24e3c35f7605aae1c76597d4ffd